modules/project/main.tf

```diff
@@ -356,3 +364,12 @@ resource "google_access_context_manager_service_perimeter_resource" "service-per
   perimeter_name = each.value
   resource       = "projects/${local.project.number}"
 }
+
+resource "google_kms_crypto_key_iam_member" "crypto_key" {
+  for_each = {
+    for service_key in local.service_encryption_key_ids_flatten : "${service_key.service}.${service_key.key}" => service_key
+  }
+  crypto_key_id = each.value.key
```

Comment:
Don't you also need to specify the project? Just asking, as I don't remember how this resource works.